L1Trigger/Configuration/python/SimL1Emulator_cff.py

```diff
@@ -167,7 +167,7 @@
 _phase2_siml1emulator.add(l1tTrackJetsExtendedEmulation)
 
 from L1Trigger.L1TTrackMatch.l1tTrackerEmuEtMiss_cfi import *
-l1tTrackerEmuEtMiss.L1VertexInputTag = ("L1VertexFinderEmulator","l1verticesEmulation")
+l1tTrackerEmuEtMiss.L1VertexInputTag = ("l1tVertexFinderEmulator","l1verticesEmulation")
```

Comment:
Is this line the same as in https://github.com/cms-sw/cmssw/pull/39401, this line of change:
https://github.com/cms-sw/cmssw/pull/39401/files#diff-c0b64ac0c0be97495049537672bca975f5be0de08e90cb72bd6000e8496bbf4cL170